api/backend/ecs/service_manager.go

```diff
@@ -57,6 +57,21 @@ func (this *ECSServiceManager) ListServices() ([]*models.Service, error) {
 	return services, nil
 }
 
+func (this *ECSServiceManager) ListServiceNames() ([]id.L0ServiceID, error) {
+	serviceARNs, err := this.ECS.Helper_ListServices(id.PREFIX)
```

Comment:
Drop any function thats `Helper_...`; just use your own that we are sure is optimal